kore/test/Test/Kore/Builtin/Builtin.hs

```diff
@@ -162,7 +162,7 @@ testEnv =
 evaluate :: TermLike Variable -> SMT (Pattern Variable)
 evaluate = evalSimplifier testEnv . TermLike.simplify
 
-evaluateT :: Trans.MonadTrans t => TermLike Variable -> t SMT (Pattern Variable)
+evaluateT :: Trans.MonadTrans t => TermLike Variable -> t (SmtT IO) (Pattern Variable)
```

Comment:
What do we need this for? The same thing happens in a few other places below.

Reply:
I totally forgot to change them back after you suggested the type alias that would make this work without running into partial application. I'll revert these!